spec/spec_helper.rb

```diff
@@ -58,6 +58,13 @@
   Capybara.register_driver :selenium do |app|
     Capybara::Selenium::Driver.new(app, browser: :chrome)
   end
+elsif ENV['WEBDRIVER'] == 'firefox'  || ENV['CI'] == true # Last part is a hack to have PR use firefox
```

Comment:
Line is too long. [102/80]<br>Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.